public/app/features/alerting/unified/rule-list/GrafanaRuleListItem.tsx

```diff
@@ -0,0 +1,63 @@
+import { GrafanaRuleGroupIdentifier } from 'app/types/unified-alerting';
+import { GrafanaPromRuleDTO, PromRuleType } from 'app/types/unified-alerting-dto';
+
+import { GrafanaRulesSource } from '../utils/datasource';
+import { totalFromStats } from '../utils/ruleStats';
+import { prometheusRuleType } from '../utils/rules';
+import { createRelativeUrl } from '../utils/url';
+
+import {
+  AlertRuleListItem,
+  RecordingRuleListItem,
+  RuleListItemCommonProps,
+  UnknownRuleListItem,
+} from './components/AlertRuleListItem';
+import { RuleActionsButtons } from './components/RuleActionsButtons.V2';
+import { RuleOperation } from './components/RuleListIcon';
+
+interface GrafanaRuleListItemProps {
+  rule: GrafanaPromRuleDTO;
+  groupIdentifier: GrafanaRuleGroupIdentifier;
+  namespaceName: string;
+  operation?: RuleOperation;
+}
+
+export function GrafanaRuleListItem({ rule, groupIdentifier, namespaceName, operation }: GrafanaRuleListItemProps) {
+  const { name, uid, labels, provenance } = rule;
+
+  const commonProps: RuleListItemCommonProps = {
```

Comment:
blocker: this needs `showLocation: false` if we're using it in the context of the "grouped" view, otherwise we would show the location twice (and we already had a PR fixing this) :)